src/app/component/ExpandableContent/ExpandableContent.component.js

```diff
@@ -25,6 +25,11 @@ export class ExpandableContent extends PureComponent {
         subHeading: PropTypes.string,
         children: ChildrenType.isRequired,
         mix: MixType.isRequired,
+        scrollToPosition: PropTypes.func,
+        productReviewsRef: PropTypes.oneOfType([
```

Comment:
`ExpandableContent` logic is NOT related to `ProductReviews` logic!

Consider adding an option in the `ExpandableContent` component which is going to scroll to the shown element (maybe with additional argument to subtract the covering elements).

Try encapsulating the scrolling functionality in the component which utilises it!